Rakefile

```diff
@@ -91,7 +91,7 @@ end
 
 desc "Build docs and install to Xcode"
 task :docs do
-  system("appledoc --keep-intermediate-files --project-name CDTDatastore --project-company Cloudant -o build/docs --company-id com.cloudant -i Classes/vendor -i Classes/common/touchdb Classes/")
+  system("appledoc --keep-intermediate-files --project-name CDTDatastore --project-company Cloudant -o build/docs --company-id com.cloudant -i CDTDatastore/vendor/ -i CDTDatastore/touchdb/ CDTDatastore")
```

Comment:
Would it be worth calling `rake docs` from our Jenkinsfile so we fail the build if we drift from a working command again?

Reply:
@ricellis I think that's a great idea, are you OK if I add it in this PR?

Reply:
yeah

Reply:
@ricellis Added `macosBuildDocs` step to `rake docs` in e581dee.